src/common/util/random.js

```diff
@@ -0,0 +1,113 @@
+/*globals define*/
+/*jshint node: true, browser: true, bitwise: false*/
+
+/**
+ * @author kecso / https://github.com/kecso
+ *
+ * collection of functions that uses random Numbers in WebGME
+ */
+
+define(['./chance.min'], function (ChanceJs) {
```

Comment:
This file should have a random.spec.js.
Try things like generateRelid when object contains different keys.